srml/staking/src/lib.rs

```diff
@@ -770,6 +771,8 @@ impl<T: Trait> OnSessionChange<T::Moment> for Module<T> {
 }
 
 impl<T: Trait> EnsureAccountLiquid<T::AccountId, BalanceOf<T>> for Module<T> {
+	// TODO: Consider replacing uses of this function in favor for ensure_account_can_withdraw
```

Comment:
It remains here on purpose. Contracts module cannot be changed in order to use the finer-grained function.

Reply:
If it remains on purpose, we should create an issue for it :)